api/Navigator.json

```diff
@@ -2459,6 +2459,62 @@
             "standard_track": true,
             "deprecated": true
           }
+        },
+        "returns_plugin_type": {
+          "__compat": {
+            "description": "Returns mime types from plugins rather than hard coded PDF values",
+            "support": {
+              "chrome": {
+                "version_removed": "94",
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_removed": "94",
+                "version_added": "18"
+              },
+              "edge": {
+                "version_removed": "94",
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_removed": "99",
+                "version_added": "1"
+              },
+              "firefox_android": {
+                "version_removed": "99",
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "4"
+              },
+              "opera": {
+                "version_removed": "80",
+                "version_added": "≤12.1"
+              },
+              "opera_android": {
+                "version_removed": "66",
+                "version_added": "≤12.1"
+              },
+              "safari": {
+                "version_added": "1"
+              },
+              "safari_ios": {
+                "version_added": "1"
+              },
+              "samsunginternet_android": {
+                "version_added": "1.0"
+              },
+              "webview_android": {
+                "version_removed": "94",
+                "version_added": "1"
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
```

Comment:
Status ought to be:

* Experimental: false (weird case of a single implementation, but an old one that's unlikely to change)
* Standard track: false
* Deprecated: true

Reply:
Fixed in #15361